src/expandableCalendar/index.js

```diff
@@ -345,20 +343,26 @@ class ExpandableCalendar extends Component {
 
   onPressArrowLeft = () => {
     this.scrollPage(false);
-  }
+  };
   onPressArrowRight = () => {
     this.scrollPage(true);
-  }
+  };
 
-  onDayPress = (value) => { // {year: 2019, month: 4, day: 22, timestamp: 1555977600000, dateString: "2019-04-23"}
+  onDayPress = (value) => {
+    // {year: 2019, month: 4, day: 22, timestamp: 1555977600000, dateString: "2019-04-23"}
     _.invoke(this.props.context, 'setDate', value.dateString, UPDATE_SOURCES.DAY_PRESS);
 
-    setTimeout(() => { // to allows setDate to be completed
+    if (this.props.onCalendarToggled) {
```

Comment:
I'm not sure that's the right place for it

Reply:
duplicated logic, will be removed